docs/rules/prefer-object-has-own.md

```diff
@@ -0,0 +1,49 @@
+# Prefer `Object.hasOwn(…)` over `Object.prototype.hasOwnProperty.call(…)` (prefer-object-has-own)
```

Comment:
```suggestion
# Prefer `Object.hasOwn()` over `Object.prototype.hasOwnProperty.call()` (prefer-object-has-own)
```

I think it looks better without the dots. In documentation for other rules, function calls also have just `()`.